sdk/samples/iot/generate_certificate.ps1

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
```

Comment:
Also on top of this please do a `Set-StrictMode on` or whatever syntax is.  This will help catch errors now and in future for us.

https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/set-strictmode?view=powershell-7#:~:text=Description%20The%20Set-StrictMode%20cmdlet%20configures%20strict%20mode%20for,or%20script%20block%20violates%20basic%20best-practice%20coding%20rules.

Reply:
Verbiage looks like its:
```powershell
Set-StrictMode -Version 3.0
```
where the number can be a certain version. I set it to 3.0